packages/flutter/test/widgets/layout_builder_test.dart

```diff
@@ -587,4 +587,41 @@ void main() {
     await tester.pump();
     expect(hitCounts, const <int> [0, 0, 0]);
   });
+
+  testWidgets('LayoutBuilder calculates baseline from child', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      Directionality(
```

Comment:
Is the directionality widget necessary?

Reply:
It is needed to render the widgets, otherwise it fails with:

`The following assertion was thrown during performLayout():
Horizontal RenderFlex with multiple children has a null textDirection, so the layout order is
undefined.
'package:flutter/src/rendering/flex.dart':
Failed assertion: line 439 pos 18: 'textDirection != null'`

Reply:
Ah gotcha, sounds good!